package/MDAnalysis/core/topologyattrs.py

```diff
@@ -1748,24 +1844,17 @@ def shape_parameter(group, wrap=False):
            Renamed `pbc` kwarg to `wrap`. `pbc` is still accepted but
            is deprecated and will be removed in version 3.0.
            Superfluous kwargs were removed.
+        .. versionchanged:: 2.5.0
+           Added use of gyration_moments for per residue quantities
```

Comment:
s this important for the user to know how the quantities are computed? I'd assume the more important message is what you have in the CHANGELOG: since 2.5.0 you can calculate asphericity for any `compound`.

Reply:
Does the change I made clarify?